plugin.video.mytv_bg/resources/language/English/strings.xml

```diff
@@ -1,23 +1,23 @@
-<?xml version="1.0" encoding="utf-8" standalone="yes"?>
-<strings>
-    <!-- General entries -->
-    <string id="30000">Page</string>
-    <string id="30001">Search</string>    
-    <string id="30002">Play</string>
-	<string id="30003">Error</string>
-	<string id="30004">Server returned no data! Please try again.</string>
-	<string id="30005">Update your account in plugin settings!</string>
-	<string id="30006">Select quality</string>
-	
-    
-    <!-- Login information entries -->
-    <string id="30200">Email</string>
-    <string id="30210">Password</string>
-
-    <!-- Server data configuration -->
-    <string id="5000">API URL</string>
-    
-    <!-- Settings -->
-    <string id="30010">Force ViewMode to Thumbnail</string>
-    <string id="30011">Update container on page switch</string>
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<strings>

```

Comment:
It would be cool if you could adapt the strings.xml to strings.po at some point in the future. See here: http://kodi.wiki/view/Language_support